### DISCUSSION TITLE
Having issues with new Image component and assetPrefix

### DISCUSSION THREAD

**Discussion by thors1982** [vercel/next.js#19833] [Help]
We are having an issue with the image component, we currently have an assetPrefix set to a subfolder so all our _next assets are loaded from a subfolder.
 
Example
```
assetPrefix: 'sub_folder_goes_here',
```
 
And our files are loading perfect from the subfolder set (example below)
```
https://mydomain.com/sub_folder_goes_here/_next/static/chunks/1.js
```
 
But with the new `Image` component, all our images are trying to load from the domain without the prefix (example below)
 
```
https://mydomain.com/_next/image?url=%2Fsub_folder_goes_here%2Fimages%2Frtb1-added.png&w=900&q=75
```
*NOTE: the subfolder is in the query parameter instead of the path
 
Which isn't working becuase it can't find the next folder.
 
We found a workaround that seems super hacky... which is setting the path like below
 
```
path: [`/sub_folder_goes_here/_next/image`],
```

Is there another way? Or is this a possible bug?

**Comment by foreseaz** [comment]
Hi, I think this is a bug and may I know the details of the workaround? If there is no way to fix it from NextJS side, I'm even thinking rewrite the `/_next` to `/basePath/_next` with AWS Lambda, which is no a good idea of course.

**Comment by lucasalmeida92** [comment]
I don't know if you guys did the same, but I was getting the same error because my `public` folder was inside `src` folder.
So I just put it in root folder and it worked!
